### DISCUSSION TITLE
Entries lost after restoring from backup

### DISCUSSION THREAD

**Discussion by Akshunhiro** [dani-garcia/vaultwarden#4039] [Q&A] [answered]
Hi all, I'm running the container on unRAID and unfortunately had a drive fail. Wasn't too concerned as I had backups of all my containers but Vaultwarden was nearly a week out. Again, wasn't too concerned as the apps and extensions should sync back to the server once it's back online. Except they didn't. Any entries during that time were lost. Completely gone. Am I wrong in assuming entries would sync back once connection to the server is restored? I'm on v1.29.2

**Comment by BlackDex** [comment]
Did you tried to run a setting your self
I actually don't know if this works or should work.
I think it worked in the past, since i did test it. But not with a week offline. But this was a long time ago.

**Reply by Akshunhiro** [reply]
I didn't force a resync if that's what you mean. Wasn't aware I needed to make additional changes after restoring the backup. Even though the backup was nearly a week old, the server itself was offline for just over 24 hours. Vaults loaded fine after logging back in on apps and extensions so tokens must've still been valid. Just lost entries crated since the most recent backup. I did see a similar issue in [this thread](https://github.com/dani-garcia/vaultwarden/discussions/3482) but no definitive answer. I guess I could try taking the container offline, creating an entry in the browser, wait an hour or so and start the container back up to see if that new test entry remains.

**Comment by jjlin** [comment, marked as answer]
The server's copy of the vault is the source of truth, and the clients aren't designed to be able to sync the difference back to the server (this is related to the lack of offline editing, which is a longstanding feature request that Bitwarden hasn't implemented yet). You'd have to export the vault on the client first and figure out which entries are new or have since been modified.

**Reply by Akshunhiro** [reply]
Ah, I see. Thanks for clearing that up!
